conf.js

```diff
@@ -0,0 +1,23 @@
+// conf.js
+exports.config = {
+  framework: 'jasmine',
+  directConnect: true, // enabled, when used without webdriver-manager
+  seleniumAddress: 'http://localhost:4444/wd/hub',
+  specs: ['test/**/*.spec.js'],
+  capabilities: {
+    browserName: 'firefox',
+    'moz:firefoxOptions': {
+      args: ['--safe-mode'],
```

Comment:
why safe mode?

Reply:
This is an example given by protractor.
From what I saw,  `--safe-mode` reduce the tests running inconsistently and increase the execution speed.